npm/react/plugins/utils/get-cypress-folders.js

```diff
@@ -0,0 +1,21 @@
+// @ts-check
+function getCypressFolders (config) {
+  const folders = []
+
+  // user can disable folders, so check first
+  if (config.componentFolder) {
+    folders.push(config.componentFolder)
+  }
+
+  if (config.fixturesFolder) {
+    folders.push(config.fixturesFolder)
+  }
+
+  if (config.integrationFolder) {
+    folders.push(config.integrationFolder)
```

Comment:
Do we need an integration folder for component testing? Adding a new folder for compilation increases the initial build time exponentially

Reply:
the same was for `cra-v3` plugin but I guess it should be fine if we exclude it